metadata-ingestion/docs/transformer/dataset_transformer.md

```diff
@@ -1632,4 +1632,24 @@ After running `datahub ingest -c <path_to_recipe>`, our MCEs will now have the f
 ],
 ```
 
+### Using this in the remote executor (DataHub Cloud only)
```

Comment:
This should probably be in https://datahubproject.io/docs/metadata-ingestion/docs/transformer/intro/#provided-transformers under a new section about custom transformers for DataHub Cloud.